### PR TITLE
S3 BUCKET HARVEST

### PR DESCRIPTION
**Pull request by tim-boost**

### CODE REVIEW

File: Gemfile.lock [hunk 1]
```diff
@@ -96,6 +97,703 @@ GEM
...
+    aws-sdk-cloudformation (1.14.0)
+      aws-sdk-core (~> 3, >= 3.39.0)
+      aws-sigv4 (~> 1.0)
+    aws-sdk-cloudfront (1.11.0)
+      aws-sdk-core (~> 3, >= 3.39.0)
+      aws-sigv4 (~> 1.0)
+    aws-sdk-cloudhsm (1.9.0)
+      aws-sdk-core (~> 3, >= 3.39.0)
+      aws-sigv4 (~> 1.0)
+    aws-sdk-cloudhsmv2 (1.9.0)
```
Comment (theomarkkuspaul) [inline Gemfile.lock]: is there a way to add just the aws s3 client rather than all of aws?

File: Gemfile [hunk 2]
```diff
@@ -23,8 +23,9 @@ gem 'rails', '5.2.1'
 gem 'responders'
 gem 'sidekiq', '= 5.2.3'
 gem 'sinatra', :require => nil
-gem 'supplejack_common', git: 'https://github.com/DigitalNZ/supplejack_common.git', tag: 'v2.5'
+gem 'supplejack_common', git: 'https://github.com/DigitalNZ/supplejack_common.git', branch: 'tw/s3-bucket-emu-harvest'
```
Comment (Reinkaos) [inline Gemfile]: You'll need to change this to the new tag of SJ common before you merge the worker.

File: Gemfile.lock (hunk 1 above)
Comment (theomarkkuspaul) [reply Gemfile.lock]: any news on this?

Comment (theomarkkuspaul) [reply Gemfile.lock]: @tim-boost

Comment (tim-boost) [reply Gemfile.lock]: yeah the current change only includes the aws-sdk-s3 gem, there are some dependencies but not as many as before

**Comment by tim-boost** [comment]
> quick comment on the aws gem in the `Gemfile.lock`

Yeah those are dependencies of the 'aws-sdk-s3' gem not as many as using the 'aws-sdk' gem which was previously used, not much I can really do here

**Pull request closed without merging**